### PR TITLE
Milestone 4 : Add a basic UI for interaction

### PR DESCRIPTION
**Pull request by asharanjith**
In this PR

- [x] Create an app.rb file that will serve as console app entry-point
- [x] Create main.rb file to define the entry point
- [x] Follow git flow

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+      print 'Has parent permission? [Y/N]: '
+      parent_permission = gets.chomp
+      parent_permission = parent_permission.downcase == 'y'
+
+      person = Student.new(age, name, parent_permission: parent_permission)
+    else
+      print 'Specialization: '
+      specialization = gets.chomp
+
+      person = Teacher.new(age, specialization, name)
```
Comment (tugsem) [inline app.rb:57]: - [ ] When we create teacher, we see specialization not the age. Kindly fix this. 
image

File: main.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    app.create_person
+  when 4
+    app.create_book
+  when 5
+    app.create_rental
+  when 6
+    app.list_rentals
+  end
+end
+
```
Comment (tugsem) [inline main.rb:30]: - [ ] Kindly show this text before exit when user choose option '7'. 
image

**Review by tugsem — CHANGES_REQUESTED** [review]
##  STATUS: REQUIRED CHANGES ♻️
Hi @asharanjith , 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights 🚀 
✔️correct gitflow
✔️linters pass
✔️professional documentation

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (asharanjith) [reply app.rb:57]: @tugsem I tried many ways to do the same. Below is the screenshot of the video of the project requirement. Can you please confirm whether it is needed?
cr

**Review by KayLemba — APPROVED** [review]
[# STATUS: APPROVED :green_circle: :clap: 

Hello  @asharanjith , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: aaa
Congratulations! 🎉 

HIGHLIGHTS:

- [x]  Your repo is professional 
- [x] Descriptive P.R
- [x] Proffessional README








 








Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 



well-1

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._]

**Pull request merged**